app/gui/src/project-view/components/GraphEditor/GraphNode.vue

```diff
@@ -464,8 +478,8 @@ const showMenuAt = ref<{ x: number; y: number }>()
     :class="nodeClass"
     :data-node-id="nodeId"
     @pointerdown.stop
-    @pointerenter="((nodeHovered = true), updateNodeHover($event))"
-    @pointerleave="((nodeHovered = false), updateNodeHover(undefined))"
+    @pointerenter="(graph.setNodeHovered(nodeId, true), updateNodeHover($event))"
```

Comment:
Perhaps `setNodeHovered` should be just a part of `updateNodeHover`? I think I see these two always called together.

Reply:
`updateNodeHover` is used on its own once, though, on pointer move.